_posts/2020-05-01-issue-1.md

```diff
@@ -59,33 +59,33 @@ Two songs this week:
 
 #### [Iceage - Lockdown Blues](https://iceage.bandcamp.com/track/lockdown-blues)
 
-_Your river flows again, my love_  
-_I know you've been choked_  
-_A loosened clamp around your beak_  
+_Your river flows again, my love_
+_I know you've been choked_
+_A loosened clamp around your beak_
 _While crowds and parties mock the weak_
 
-_A sudden fatal tenant_  
-_Seizes room_  
-_Didn't grasp what I had until_  
+_A sudden fatal tenant_
+_Seizes room_
+_Didn't grasp what I had until_
 _It was robbed absent too soon_
 
 Says the band, “All profits from Bandcamp will proceed directly to Médecins Sans Frontières who right now are doing their best to provide aid where it so urgently is needed the most.”
 
-##### [GmacCash - On Strike (Shot By DaPoundMedia)](https://www.youtube.com/watch?v=uJOHvQ9cL38)
+#### [GmacCash - On Strike (Shot By DaPoundMedia)](https://www.youtube.com/watch?v=uJOHvQ9cL38)
```

Comment:
Fix is here. (this was h5 instead of h4)

Sorry, my edit trims trailing spaces automatically. 😬

Reply:
Pretty sure that was my bug. Thanks for fixing.

(And thanks for trimming the whitespace, too. I usually set my editors for that too, heh.)